### DISCUSSION TITLE
Link component gives href attribute error for not localized paths

### DISCUSSION THREAD

**Discussion by emirxmertoglu** [amannn/next-intl#1512] [Q&A]
I have some pages unlocalized or some external links, when I use Link components for those paths it gives a type error on the href attribute like this,
![image](https://github.com/user-attachments/assets/03a028b7-b6dd-454e-8640-61ca4ebaf33c)

And also I have localized pathnames like below,
![image](https://github.com/user-attachments/assets/6941755b-c419-434c-b7f0-7156321db32e)

What should I do, any idea?
I work with
```ts
    "next": "^15.0.2",
    "next-intl": "^3.24.0",
```

**Comment by amannn** [comment]
See: [How can I compose the link with its href prop?](https://next-intl-docs.vercel.app/docs/routing/navigation#link-composition)
